.github/release.yml

```diff
@@ -0,0 +1,24 @@
+changelog:
+    exclude:
+      authors:
+        - dependabot[bot]
+    categories:
+      - title: Enhancements
+        labels:
+          - enhancement
+      - title: Bug fixes
+        labels:
+          - bug
+      - title: Documentation
+        labels:
+          - documentation
+      - title: Examples
+        labels:
+          - examples
+      - title: CI/CD
+        labels:
+          - CI/CD
+      - title: Maintenance
+        labels:
+          - maintenance
+          - dependencies
```

Comment:
Is it a logical 'OR' or a logical 'AND'?

Reply:
It's an OR